src/internal/constants.ts

```diff
@@ -6,6 +6,11 @@ export const version = '1.6.0';
  */
 export const defaultSDKVersionForCompatCheck = '1.6.0';
 
+/**
+ * This is the SDK version when captureImage API is supported on mobile.
+ */
+export const captureImageMobileSupportVersion = '1.7.0';
```

Comment:
Can we directly consume '1.7.0' in captureImage API without declaring a constant for this. Doesn't look like it will be used anywhere else and going forward when we introduce new APIs, these constants will also grow in number.

Reply:
Done. Moved this to media.ts